### ISSUE TITLE
Error starting sf.exe in Windows 7 x64

### ISSUE THREAD

**Issue by raveydaveygravy** [smicallef/spiderfoot#124]
I extracted the contents of SpiderFoot-2.8.0-w32.zip to c:\spiderfoot-2.8.0\

I see the error below when attempting to run sf.exe at cmd.exe (with and without admin privileges).

Any ideas how I can fix this please?

```
PS C:\spiderfoot-2.8.0> .\sf.exe 0.0.0.0:9999
Traceback (most recent call last):
  File "sf.py", line 54, in <module>
  File "zipextimporter.pyc", line 82, in load_module
  File "sflib.pyc", line 26, in <module>
  File "zipextimporter.pyc", line 82, in load_module
  File "ssl.pyc", line 98, in <module>
  File "zipextimporter.pyc", line 98, in load_module
ImportError: MemoryLoadLibrary failed loading _ssl.pyd
```

**Comment by abhiinos111abhi** [comment]
same problem i faced

**Comment by dtfew** [comment]
Same problem here win 8.1 x 64 . any updates?

**Comment by smicallef** [comment]
I've found the issue causing this, it will be addressed in 2.9 coming out in a week or so.

**Comment by abhiinos111abhi** [comment]
i just used old version 2.7.1 working good

**Comment by smicallef** [comment]
Should now be fixed in 2.9.0.

**Comment by dtfew** [comment]
all good now thanks!
